framework/src/iotbus/iotbus_gpio.c

```diff
@@ -155,6 +155,58 @@ int iotbus_gpio_set_direction(iotbus_gpio_context_h dev, iotbus_gpio_direction_e
 	return IOTBUS_ERROR_NONE;
 }
 
+/**
+ * @brief Registers signal for current process based on rising or falling edge on the gpio line
+ */
+int iotbus_gpio_register_signal(iotbus_gpio_context_h dev, iotbus_gpio_edge_e edge)
```

Comment:
Don't mix different kind of changes. This is an implementation of framework, not testcase. Please split commit.

Reply:
Ok I split it